packages/browsers/src/browsers/firefox.ts

```diff
@@ -38,26 +38,18 @@ export function resolveDownloadUrl(
   return `${baseUrl}/${archive(platform, revision)}`;
 }
 
-export function executablePath(
+export function relativeExecutablePath(
   platform: BrowserPlatform,
-  revision: string,
-  basePath = ''
+  _revision: string
 ): string {
-  const browserPath = path.join(basePath, `${platform}-${revision}`);
   switch (platform) {
     case BrowserPlatform.MAC_ARM:
     case BrowserPlatform.MAC:
-      return path.join(
-        browserPath,
-        'Firefox Nightly.app',
-        'Contents',
-        'MacOS',
-        'firefox'
-      );
+      return path.join('Firefox Nightly.app', 'Contents', 'MacOS', 'firefox');
     case BrowserPlatform.LINUX:
-      return path.join(browserPath, 'firefox', 'firefox');
+      return path.join('firefox', 'firefox');
```

Comment:
Why do we put it in a subfolder for firefox and not for chrome?

Reply:
That's how the Firefox archive is structured I believe:

```
~/.cache/puppeteer/firefox/linux-112.0a1/firefox/firefox
```